test/Window/Context.test.cpp

```diff
@@ -33,6 +33,28 @@ TEST_CASE("[Window] sf::Context", runDisplayTests())
         CHECK(context.getActiveContextId() != 0);
     }
 
+    SECTION("Move semantics")
+    {
+        SECTION("Construction")
+        {
+            sf::Context       movedContext;
+            const sf::Context context(std::move(movedContext));
+            CHECK(context.getSettings().majorVersion > 0);
+            CHECK(context.getActiveContext() == &context);
+            CHECK(context.getActiveContextId() != 0);
```

Comment:
Those are static methods.
```suggestion
            CHECK(sf::Context::getActiveContext() == &context);
            CHECK(sf::Context::getActiveContextId() != 0);
```
Also applies to other test cases.